discovery-infra/resources/sno_manifest.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
I think it's not required anymore. @eranco74 ?

Reply:
@eranco74 will soon change it with the new payload